src/components/BadgeSlider/DesktopSlider.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react';
+import Badge from 'components/Badge';
+import rightArrowIcon from 'static/icons/right-arrow.svg';
+import leftArrowIcon from 'static/icons/left-arrow.svg';
+import { IBadge } from '.';
+import ArrowButton from './ArrowButton';
+import { chunkArray } from 'utils/array';
+
+const ITEMS_PER_PAGE = 3;
+
+interface Props {
+  items: IBadge[];
+}
+
+const DesktopSlider: React.FC<Props> = ({ items }) => {
+  const [currentPage, setCurrentPage] = useState(1);
```

Comment:
@arathjz while I think this should be easy enough, is there a library we can use for this?

Reply:
I've heard about this one https://www.react-spring.io. We could give it a try

Reply:
react-spring is more for animations I think @orlando suggest in the slider itself something like https://github.com/brainhubeu/react-carousel so we add the dependency and then:

```
<Carousel plugins={['arrows']}>
    <img src={imageOne} />
    <img src={imageTwo} />
    <img src={imageThree} />
</Carousel>
```

_Maybe other with more starts but I guess the point is not having more code within the project for this._